src/lib/Libpython/pbs_python_external.c

```diff
@@ -186,11 +193,45 @@ pbs_python_ext_start_interpreter(struct python_interpreter_data *interp_data)
 	}
 
 	/*
-	 * arg '1' means to not skip init of signals
-	 * we want signals to propagate to the executing
-	 * Python script to be able to interrupt it
-	 */
+	* arg '1' means to not skip init of signals
+	* we want signals to propagate to the executing
+	* Python script to be able to interrupt it
+	*/
```

Comment:
I think we usually add a space before * so that all * align vertically.

Reply:
changed